### DISCUSSION TITLE
希望直播也添加默认播放器模式

### DISCUSSION THREAD

**Discussion by Gahotx** [the1812/Bilibili-Evolved#3079] [功能建议 / Ideas]
如题，希望直播也能一进直播间就自动网页全屏或者全屏

**Comment by Geeyun-JY3** [comment]
请先善用搜索。重复：#965

**Reply by Gahotx** [reply]
![image](https://user-images.githubusercontent.com/53534938/156613972-6259723b-cf15-4669-98d0-77a03bbe3505.png)
emmm，在讨论区简单搜了下没看到

**Reply by Geeyun-JY3** [reply]
非要说的话讨论区也有：#1994、#2439

**Reply by Geeyun-JY3** [reply]
还是有不少功能建议在 issue 区的，也不知道还转不转移。
![Screenshot.jpg](https://user-images.githubusercontent.com/49396421/156716151-2e835f40-d303-46ca-9ca6-69672a3d1df0.jpg)

**Reply by Gahotx** [reply]
行了行了，知道了，去看其他人的讨论吧
